plugins/base/src/test/kotlin/signatures/SignatureUtils.kt

```diff
@@ -0,0 +1,12 @@
+package signatures
+
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Element
+import utils.TestOutputWriter
+
+fun TestOutputWriter.renderedContent(path: String = "root/example.html") =
+    contents.getValue(path).let { Jsoup.parse(it) }.select("#content")
+        .single()
+
+fun Element.signature() = select("div.symbol.monospace")
+fun Element.firstSignature() = signature().first()
```

Comment:
Hey small tip: 
You might want to enable the intelliJ setting: 
"New line on save" to ensure that you always end files with a newline ☺️

Reply:
Ok, will change, thanks